tests/functional/test_direct_edit.py

```diff
@@ -210,6 +210,77 @@ def extract_edit_info(_):
         assert file.is_file()
 
 
+def test_office_temporary_files(manager_factory, obj_factory):
+    """
+    Software from the office suite may leave temporary file behind them.
+    This test goal is to ensure that these files are not removed at startup
+    nor uploaded to the remote by the _cleanup method.
+    Uploading a temporary file may result in data loss on the remote.
```

Comment:
```suggestion
    There are 2 situations we need to prevent:

    1. Softwares LibreOffice or MS Office may leave temporary files behind them.
    2. Someone could intentionally put a bad file in the good folder.

    Both will corrupt the remote document (data loss) at the next startup.

    The test goal is to ensure that these files are not just ignored.
```